homeassistant/components/nut/__init__.py

```diff
@@ -79,6 +81,22 @@ async def async_update_data() -> dict[str, str]:
     await coordinator.async_config_entry_first_refresh()
     status = coordinator.data
 
+    all_device_supported_commands = data.list_commands() or {}
+    supported_commands: set[str] = set()
+
+    for [
```

Comment:
The [] here can be dropped

Reply:
I removed the `[]` but after saving the file but then`()` are automatically added; I left the latter. Please let me know if there is a better way to do it. Thanks!

Reply:
() is faster than [] so all fine there